app/src/main/res/values/strings.xml

```diff
@@ -12,10 +12,17 @@
     <string name="event_card_date">Event Date :</string>
 
     <!--profile details-->
+    <string name="email_head">Email</string>
```

Comment:
email_head ?

Reply:
the email hint actually. will change the name asap